plugin/src/main/scala/com/lightbend/paradox/sbt/ParadoxPlugin.scala

```diff
@@ -71,7 +71,7 @@ object ParadoxPlugin extends AutoPlugin {
     paradoxProperties ++= linkProperties(scalaVersion.value, apiURL.value, scmInfo.value),
 
     sourceDirectory in paradox := sourceDirectory.value / "paradox",
-    sourceDirectories in paradox := Seq((sourceDirectory in paradox).value),
+    sourceDirectories in paradox := Seq(Option((sourceDirectory in paradox).value), paradoxOverlayDirectory.?.value).flatten,
```

Comment:
Yes "sourceDirectory in paradox" is the main directory of the project. We just add the "paradoxOverlayDirectory" to the sourceDirectories for the current project.